src/hearings/converters/additional-instructions.converter.ts

```diff
@@ -0,0 +1,21 @@
+import {Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {State} from '../store';
+import {AnswerConverter} from './answer.converter';
+
+export class AdditionalInstructionsConverter implements AnswerConverter {
```

Comment:
Could we rename the file to additional-instructions.answer.converter and this to AdditionalInstructionsAnswerConverter? This is to consistent with other converters.

Reply:
Sure, that wasn't the naming convention when I did the work